crates/ide/src/syntax_highlighting/tags.rs

```diff
@@ -67,6 +67,8 @@ pub enum HlMod {
     Trait,
     /// Used with keywords like `async` and `await`.
     Async,
+    /// Used for items from other crates.
+    Foreign,
```

Comment:
Is there a better name here? Foreight might be confused with FFI. Perhaps "dependency"?

Reply:
What about `External`?

Reply:
I wonder if `library` would fit? Since vscode pushes `defaultLibrary` for `symbols that are part of the standard library`.

Reply:
Sounds good! I’ll change it to `library`.